docker_images/sklearn/tests/generators/iris.py

```diff
@@ -26,9 +28,15 @@ def push_repo(repo_name, local_repo):
     print(f"Pushing {repo_id}")
 
     client = HfApi()
-    client.delete_repo(repo_id, token=token)
+    try:
+        client.delete_repo(repo_id, token=token)
+    except RepositoryNotFoundError:
```

Comment:
If we can't import this exception from a public path from `huggingface_hub`, I rather catch `Exception` instead.

Or to check if the repo exists explicitly

Reply:
I'm changing this to `from huggingface_hub.utils import RepositoryNotFoundError`.